js/count-chars.js

```diff
@@ -12,24 +12,32 @@
 
 var aiosp_title_extra = parseInt( aioseop_count_chars.aiosp_title_extra );
 
-jQuery( document ).ready( function() {
-	aioseopInitCounting();
-});
+jQuery( document ).ready(
+	function() {
+			aioseopInitCounting();
+	}
+);
 
 /**
  * Preview Snippet
  *
  * @since ?
  * @since 2.9.2 Move from PHP value to JS file
  */
-jQuery(document).ready( function() {
-	jQuery("#aiosp_title_wrapper").bind("input", function() {
-		jQuery("#aiosp_snippet_title").text(jQuery("#aiosp_title_wrapper input").val().replace(/<(?:.|\n)*?>/gm, ""));
-	});
-	jQuery("#aiosp_description_wrapper").bind("input", function() {
-		jQuery("#aioseop_snippet_description").text(jQuery("#aiosp_description_wrapper textarea").val().replace(/<(?:.|\n)*?>/gm, ""));
-	});
-});
+jQuery( document ).ready(
```

Comment:
This also doesn't need to be broken down. Everyone generally expects `jQuery( document ).ready( function() {`

I also don't think it is ideal to indent so much; which is a result from the change in coding style. Just ends up looking like a lot of nested code. 

...and it looks like this occurs for every line with jQuery(element).do_something( function() {